auditbeat/module/auditd/show_linux.go

```diff
@@ -80,19 +81,19 @@ func showAuditdRules() error {
 	for idx, raw := range rules {
 		r, err := rule.ToCommandLine(raw, !dontResolveIDs)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Error decoding rule %d: %v\n", idx, err)
-			fmt.Fprintf(os.Stderr, "Raw dump: <<<%v>>>\n", raw)
+			fmt.Fprintf(stderr, "Error decoding rule %d: %v\n", idx, err)
+			fmt.Fprintf(stderr, "Raw dump: <<<%v>>>\n", raw)
 		}
-		fmt.Println(r)
+		fmt.Fprintln(stdout, r)
 	}
 
 	if !noOutputIfEmpty && len(rules) == 0 {
-		fmt.Println("No rules")
+		fmt.Fprintln(stdout, "No rules")
```

Comment:
It makes me very sad that this has to happen; I think forbidigo is a bad linter.

In the same situation I have used `os.Stdout.WriteString("...\n")`.

Reply:
I think allowing the author to use discretion would be better. 

But if I were writing this from scratch I would have written it like this anyway (except I would have used `RunE` instead of os.Exit). This allows for a bit of testing to occur by injecting your own output/error streams. Then you can validate the output. For example https://github.com/andrewkroh/go-examples/blob/f4f2b8b3e96f2cdfcf4b05af50039ebb9631144a/elastic-package-changelog/cmd/add_next_test.go#L18-L29.